examples/multilang/__main__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016-2018, Pulumi Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pulumi
+from pulumi import ResourceOptions
+from pulumi_aws import ec2
+
+# TODO - this should go inside `pulumi_aws`.
+pulumi.runtime.register_proxy_constructor("aws:ec2/securityGroup:SecurityGroup", lambda name, opts: ec2.SecurityGroup(name, ResourceOptions(**opts)));
```

Comment:
does SecuityGroup not have args that are needed?  or is the idea that the opts supply enough info for SecurityGroup to hydrate itself?